com.google.AndroidStudio.appdata.xml

```diff
@@ -25,9 +25,27 @@
 		</ul>
 		<p>NOTE: This wrapper is not verified by, affiliated with, or supported by Google.</p>
 	</description>
+  <categories>
+    <category>Development</category>
+    <category>IDE</category>
+  </categories>
 	<icon type="remote" height="256" width="256">https://1.bp.blogspot.com/-DAAJWGccTCQ/We_MGiMuSFI/AAAAAAAAEtk/5_COppUTDusIIx725jENI3fd9rqlRl0gQCLcBGAs/s256/image8.png</icon>
 	<icon type="remote" height="128" width="128">https://1.bp.blogspot.com/-DAAJWGccTCQ/We_MGiMuSFI/AAAAAAAAEtk/5_COppUTDusIIx725jENI3fd9rqlRl0gQCLcBGAs/s128/image8.png</icon>
 	<icon type="remote" height="64" width="64">https://1.bp.blogspot.com/-DAAJWGccTCQ/We_MGiMuSFI/AAAAAAAAEtk/5_COppUTDusIIx725jENI3fd9rqlRl0gQCLcBGAs/s64/image8.png</icon>
```

Comment:
Can you remove this too? An icon is installed here https://github.com/flathub/com.google.AndroidStudio/blob/f342ee897d68575c80c1bcbfcaac01825ebe1544/com.google.AndroidStudio.json#L30